stanford_profile_helper.module

```diff
@@ -1015,7 +1015,7 @@ function stanford_profile_helper_form_media_embeddable_add_form_alter(&$form, Fo
       ->getConfiguration();
     $source_field = $source_config['source_field'];
     $new_desc = 'Allowed providers: @providers. For custom embeds, please <a href="@snow_form">request support.</a>';
-    $args = $form[$source_field]['widget'][0]['value']['#description']->getArguments();
+    $args = $form[$source_field]['widget'][0]['value']['#description']['#items'][1]->getArguments();
```

Comment:
Can we move this to the function `stanford_profile_helper_field_widget_form_alter`?

Reply:
Probably?  I'll spin it back up and give it a try.

Reply:
It was possible, and I did so with the new commit.  There is a similar block of code in the `hook_form_media_library_add_form_embeddable_alter` function on line 991, but that one needs to be kept in place, since apparently it doesn't run the same hooks in the ajax media library.  I removed the other redundant instances though.